projects/observability/src/shared/components/donut/donut.ts

```diff
@@ -41,3 +42,8 @@ export const enum DonutAlignmentStyle {
   Center = 'center-alignment',
   Right = 'right-alignment'
 }
+
+export const enum DonutType {
```

Comment:
```suggestion
export const enum ChartType {
```

Reply:
Changing to `DonutChartType` since all other types/interface wrt to this component start with `Donut`